paddle/fluid/pir/dialect/operator/utils/utils.cc

```diff
@@ -74,7 +74,8 @@ const std::unordered_set<std::string> LegacyOpList = {
     paddle::onednn::dialect::LrnGradOp::name(),
 #endif
     CReduceMinOp::name(),
-    PushSparseV2Op::name()};
+    PushSparseV2Op::name(),
+    LarsMomentumOp::name()};
```

Comment:
只需要对通过PD_STRUCT_KERNEL这种类型注册kernel的算子声明legacy，lars_momentum不符合这个条件。
```suggestion
    PushSparseV2Op::name()};
```

Reply:
收到